data/transactions/logic/evalStateful_test.go

```diff
@@ -461,45 +511,71 @@ func testAppFull(t *testing.T, program []byte, gi int, aid basics.AppIndex, ep *
 	return delta
 }
 
+// testLogicRange allows for running tests against a range of avm
+// versions. Generally `start` will be the version that introduced the feature,
+// and `stop` will be 0 to indicate it should work right on up through the
+// current version.  `stop` will be an actual version number if we're confirming
+// that something STOPS working as of a particular version. Note that this does
+// *not* use different consensus versions. It is tempting to make it find the
+// lowest possible consensus version in the loop in order to support the `v` it
+// it working on.  For super confidence, one might argue this should be a nested
+// loop over all of the consensus versions that work with the `v`, from the
+// first possible, to vFuture.
+func testLogicRange(t *testing.T, start, stop int, test func(t *testing.T, ep *EvalParams, tx *transactions.Transaction, ledger *Ledger)) {
+	t.Helper()
+	if stop == 0 { // Treat 0 as current max
```

Comment:
```suggestion
	if stop <= 0 { // Treat 0 as current max
```

Reply:
Then we'll get some people using 0 and some people using -1. I'd prefer to demand consistency.

I'm open to changing it to == -1, though it'll make a big diff.